examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -378,19 +390,18 @@
     "        out_net = model(torch.from_numpy(data_agent[\"image\"]).unsqueeze(0).to(device))\n",
     "        out_pos = out_net[0].reshape(-1, 2).detach().cpu().numpy()\n",
     "        # store absolute world coordinates\n",
-    "        predicted_positions.append(out_pos + data_agent[\"centroid\"][:2])\n",
+    "        predicted_positions.append(transform_points(out_pos, data_agent[\"world_from_agent\"]))\n",
```

Comment:
I also above did not fully understand why when storing predictions we need to substract ego_centroid from the transformed coordinates - is it correct without here?

Reply:
because GT coords are stored as displacements in world space. When we transform here we get to absolute coordinates in world space.

Reply:
Got it.